풀리퀘

wolframhwang opened the pull request:
오예

wolframhwang committed:
스토리보드 모드

wolframhwang committed:
Modeling Commit

wolframhwang committed:
사인업에서 변화 델리게이트 조작

wolframhwang committed:
버튼기능 추가

wolframhwang committed:
Alamofire API 설계 진행 중

wolframhwang committed:
Key chain + Alamofire APIs

wolframhwang committed:
ViewController 추가

wolframhwang committed:
Alamofire Common API

wolframhwang committed:
main position 에서 사이드뷰 만듬

wolframhwang committed:
Table View Section 별로 Expandable할수 있게 하는 코드 (공부용)

wolframhwang committed:
main frame

wolframhwang committed:
Main View에 테이블 뷰랑 Search Bar 위치 지정

wolframhwang committed:
snap kit 추가

wolframhwang committed:
호우호우...

wolframhwang committed:
View 전환 교체, Sign In, Sign Up

wolframhwang committed:
Transition

wolframhwang committed:
고민사항

wolframhwang committed:
remove Nib in TableView

wolframhwang committed:
tab bar 추가

wolframhwang committed:
초기 시작 View를 Navigation View로 변환했습니다.
위의 아이템을 Navigation Bar를 쓰는것이 좋다고 판단했는데, 가장 큰 이유는 레이아웃의 간결함때문입니다.

wolframhwang committed:
Side Menu 활용할때 Navigation View Controller를 통한 화면 뷰 전환을 구현했습니다.

wolframhwang committed:
불필요하고 헷갈릴수 있는 MainViewTable에 사용되던 기존의 Nib을 지웠습니다.

wolframhwang committed:
Post Link

wolframhwang committed:
포스트뷰 Indicator 조정!

wolframhwang committed:
Tab Bar Naivgation View Controller 를 추가했읍니다.

wolframhwang committed:
Header 추가했고, 사이드 뷰에서 이쁘게 로그인이 가능합니다!

wolframhwang committed:
Make Side View Header on SideMenuViewController

wolframhwang committed:
2중 탭바!!

wolframhwang committed:
fix storyboard

Signed-off-by: wolframhwang <hzw94@naver.com>

wolframhwang committed:
수정사항 반영

Signed-off-by: wolframhwang <hzw94@naver.com>

wolframhwang committed:
pod

wolframhwang committed:
로그인, 회원가입 API 사용할수 있게 수정

wolframhwang committed:
화면 수정

wolframhwang committed:
SignUp에 성공하면 Alert를 사용할수 있게 구현하였습니다!

wolframhwang committed:
NotificationCenter넣어봤는데.. 필요한가?..

wolframhwang committed:
로그인 로그아웃 구현!

wolframhwang committed:
Update README.md

wolframhwang committed:
Update README.md

wolframhwang committed:
Update README.md

wolframhwang committed:
Scene 관계정립

wolframhwang committed:
Merge remote-tracking branch 'refs/remotes/origin/sb'

wolframhwang committed:
refresh

wolframhwang committed:
Alamofire API Service 수정 + 추가적인 구조체

wolframhwang committed:
로그인 및 회원가입 API에 맞는 Action 기능을 구현했습니다.

wolframhwang committed:
구조체 tokenInfo -> TokenInfo (Swift Naming Rule)

wolframhwang committed:
Renaming :
ViewController -> SingInViewController
else:
	as Swift Naming Rule

wolframhwang committed:
불필요한 ViewController 생성시의 주석 제거

wolframhwang committed:
포스트로 들어가고 갤러리로 들어가기 위해 만든 코드입니다!

wolframhwang committed:
Update README.md

unluckyjung commented:
커밋로그는 [AngularJS Commit Message Conventions](https://gist.github.com/stephenparish/9941e89d80e2bc58a153) 를 참고해서 남겨봐도 좋겟네요.

```
feat (feature)
fix (bug fix)
docs (documentation)
style (formatting, missing semi colons, …)
refactor
test (when adding missing tests)
chore (maintain)
```

> 예시  

```
feat: 포스트, 갤럭시 입장 기능 구현
refactor: ViewController 생성시의 주석 제거

// 도메인의 대한 정보도 넣고 싶은경우
feat(MainView): 포스트 입장 기능 구현
```

unluckyjung commented:
swift 문법의 경우에는 [구글 컨벤션](https://google.github.io/swift/)을 참고해보시는것도 도움이 될거에요.  
분량이 상당히 되니, 전부 다 적용하거나 읽어보기보다는 주로쓰는 문법들 위주로 한번 보는걸 추천드려요.

wolframhwang commented:
@unluckyjung 
조언 감사합니다.
commit도 그렇고 코드도 그렇고 짧은 시간 이지만 정성스럽게 짜는게 맞는데,
너무 매번 급하게 커밋하고 푸시하다보니 저렇게 됐네요
반성하고 좀더 열심히짜보도록하게습니다

wolframhwang committed:
feat: Add on RefreshToken Function

- Token을 Refresh 할수 있는 함수를 APIService에 추가하였습니다.

wolframhwang committed:
Merge remote-tracking branch 'refs/remotes/origin/sb'

wolframhwang committed:
refactoring: MainViewController에서 필요없는 주석제거

- 뷰 컨트롤러 내부의 viewDidLoad에서 불필요한 주석들을 제거했습니다.

wolframhwang committed:
refactor: viewDidLoad에서 함수 호출순서 변경

- viewDidLoad에서 사용될 super,viewDidLoad()의 위치를 변경했습니다.

wolframhwang committed:
feat: Main View 진입 시 Token 있을 경우, Refresh 진행 추가

- Main View에 진입 시  Token 있는 경우에 한하여 Refresh를 진행하게 추가했습니다, refresh 인증 또는 네트워크 접속에 실패하면 Alert를 통해 확인 가능하도록 기능을 구현했습니다.
